### PR TITLE
Add version `1.6.0` to the Release Notes

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR adds a new version `1.6.0` to the Release Notes.

### CODE REVIEW

**Comment by JuliaEvseeva** [comment]
@oleksandra-kovalenko, PTAL.

**Review by oleksandra-kovalenko — APPROVED** [review]
LGTM

File: release-notes/2020/v1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    `TenantProvider.update(tenantId)`. For single-tenant applications, the `TenantProvider` can 
+    be omitted in the options.
+    
+    ```
+    Multitenant client usage example:
+    ```
+   
+    ```java
+    
let tenantProvider = new TenantProvider(initialTenantId());
```
Comment (dmitrykuzmin) [inline release-notes/2020/v1.6.0.md:255]: Could you please delete an extra space before `let`, so it's aligned with ` ```java `?

File: release-notes/2020/v1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    
let client = spineWeb.init({
```
Comment (dmitrykuzmin) [inline release-notes/2020/v1.6.0.md:256]: Same as above.

File: release-notes/2020/v1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    		protoIndexFiles: [protoIndex],
+     		endpointUrl: endpointUrl,
+     		firebaseDatabase: firebaseDatabase,
+      		actorProvider: new ActorProvider(),
+      		tenantProvider: tenantProvider
+    	});
```
Comment (dmitrykuzmin) [inline release-notes/2020/v1.6.0.md:262]: And also an extra space at the end of this line.

**Review by dmitrykuzmin — COMMENTED** [review]
Could you please fix a couple of formatting mistakes made in the release notes?

**Comment by JuliaEvseeva** [comment]
@dmitrykuzmin, PTAL.

**Review by dmitrykuzmin — APPROVED** [review]
LGTM

**Pull request merged**